templates/taxi/car_detail.html

```diff
@@ -4,9 +4,29 @@
   <p>Manufacturer: ({{ car.manufacturer.name }}, {{ car.manufacturer.country }})</p>
   <h1>Drivers</h1>
   <hr>
-  <ul>
-  {% for driver in car.drivers.all %}
-    <li>{{ driver.username }} ({{ driver.first_name }} {{ driver.last_name }})</li>
-  {% endfor %}
-  </ul>
+  <table class="table table-striped">
+    <thead>
+    <tr>
+      <th>Car</th>
+      <th>Username</th>
+      <th>First Name</th>
+      <th>Last Name</th>
+      <th>Actions</th>
+    </tr>
+    </thead>
+    <tbody>
+    {% for driver in car.drivers.all %}
+      <tr>
+        <td>{{ car.model }}</td>
+        <td>{{ driver.username }}</td>
+        <td>{{ driver.first_name }}</td>
+        <td>{{ driver.last_name }}</td>
+        <td>
+          <a href="{% url 'taxi:car-update' car.id %}" class="btn btn-primary btn-sm">Update</a>
+          <a href="{% url 'taxi:car-delete' car.id %}" class="btn btn-danger btn-sm">Delete</a>
```

Comment:
The `{% url %}` template tag generates URLs dynamically using the URL name defined in your `urls.py` file. The parameters you pass must match the named parameters expected by the URL pattern.

In your `urls.py` file, I see:
```
path("cars/<int:pk>/update/", CarUpdateView.as_view(), name="car-update"),
path("cars/<int:pk>/delete/", CarDeleteView.as_view(), name="car-delete"),
```

However, in your template code, I see a different URL format:
```html
<a href="{% url 'taxi:car-update' car.id %}" class="btn btn-primary btn-sm">Update</a>
```

Since your URL pattern uses `<int:pk>`, you should use `pk=` in the template tag:
```html
<a href="{% url 'taxi:car-update' pk=car.id %}" class="btn btn-primary btn-sm">Update</a>
```

Ensure that the href in your template matches the URL pattern, or update the URL pattern to match the parameters used in your template.